composer.json

```diff
@@ -65,7 +65,9 @@
         "drupal/materialize": "1.x-dev",
         "drupal/schemata": "^1.0@alpha",
         "drupal/openapi": "^1.0@alpha",
-        "drupal/material_admin": "1.x-dev"
+        "drupal/material_admin": "1.x-dev",
+        "drupal/snail": "1.x-dev",
```

Comment:
@dawehner Travis failed on 5.6 on PHP out of memory exception caused by composer. If the latest 8.x-1.x gets merge into this branch, that error won't happen :)